about.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./index.css" />
+    <link rel="stylesheet" href="./about.css" />
+    <title>About Us</title>
+  </head>
+  <body>
+    <header>
+      <div class="header-line only-desktop">
+        <ul>
+          <li class="header-line-list-img">
+            <a href="https://twitter.com/AbelSeifu15"
+              ><img src="./Images/twitter.png" alt="Twitter icon"
+            /></a>
+          </li>
+          <li class="header-line-list-img">
+            <a href="https://www.linkedin.com/in/abel-seifu-184543233/"
+              ><img src="./Images/linkedin.png" alt="Linkedin icon"
+            /></a>
+          </li>
+          <li class="header-line-list">
+            <a href="#">English</a>
+          </li>
+          <li class="header-line-list">
+            <a href="#">MyPage</a>
+          </li>
+          <li class="header-line-list">
+            <a href="#">Logout</a>
+          </li>
+        </ul>
+      </div>
+      <nav>
+        <div class="only-mobile mobile-nav-bar">
+          <a href="./index.html">
+            <h2 class="logo">Ethio Global Tech</h2>
+          </a>
+          <div class="only-mobile hamburger-icon">
+            <img src="./Images/hamburger-icon.svg" alt="Hamburger icon" />
+          </div>
+          <div class="mobile-menu" id="mobile-menu">
+            <div class="mobile-menu-header">
+              <button data-close-button class="close-button">&times;</button>
+            </div>
+            <ul class="mobile-menu-list">
+              <li><a href="./index.html">Home</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Join</a></li>
+              <li><a href="#">Sponsor</a></li>
+              <li><a href="#">News</a></li>
+            </ul>
+          </div>
+        </div>
+        <div class="only-desktop desktop-nav-bar">
+          <a href="./index.html">
+            <h2 class="logo">Ethio Global Tech</h2>
+          </a>
+          <div class="menu-bar-ul">
+            <ul class="menu-bar">
+              <li><a href="./index.html">Home</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Join</a></li>
+              <li><a href="#">Sponsor</a></li>
+              <li><a href="#">News</a></li>
+              <li><a class="menu-bar-btn" href="#">Ethio Tech Campaign</a></li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <section class="headline-section">
+        <div class="headline-title-text">
+          <h2>
+              ETHIO GLOBAL TECH <br />
+              CONFRENCE <br /> 
+              2022
+            </h2>
```

Comment:
- [ ] Kindly use `h1` tag for the principal topic of the page, for the best practices of HTML. 👍🏽

Reply:
Fixed, Thank you.